src/IoTHub.Portal.Shared/Models/v1.0/UserDetailsModel.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace IoTHub.Portal.Shared.Models.v10
+{
+    //using System.Collections.Generic;
+    public class UserDetailsModel
+    {
+        public string Id { get; set; }
+        public string Email { get; set; }
+        public string Name { get; set; }
+        public string Forename { get; set; }
+        //public ICollection<AccessControlModel> AccessControls { get; set; } = new List<AccessControlModel>();
```

Comment:
Remove useless code

```suggestion
        //public ICollection<AccessControlModel> AccessControls { get; set; } = new List<AccessControlModel>();
```

Reply:
This is not useless, it prepares the 'User' class to collect the accessControl list, which has not yet been implemented. It's why i decide to comment it.